variables.tf

```diff
@@ -143,6 +143,11 @@ variable "lambda_function_timeout" {
   default     = 180
 }
 
+variable "lambda_function_environment_variables" {
+    description = "Environment variables for the lambda function"
+    default = { }
+}
```

Comment:
Too many spaces in front of `description` + `default`. 
There should be just "2" and not "4"...